apps/website/src/pages/api/auth/[...nextauth].ts

```diff
@@ -41,9 +42,17 @@ export const authOptions: NextAuthOptions = {
           access_token: true,
           refresh_token: true,
           verified_at: true,
+          scope: true,
         },
```

Comment:
You could get the linked channels here to save the second query.

```suggestion
          scope: true,
          twitchChannelBroadcaster: true,
          twitchChannelModerator: true,
        },
```